lib/scrabble.rb

```diff
@@ -1,9 +1,37 @@
 class Scrabble
 
+  attr_reader :word
+
+  def initialize
+    @word = word
+  end
+
   def score(word)
-    1
+    if @word == nil
```

Comment:
You're passing `word` as an argument here, but instead of using that you're using the instance variable that you created up in initialize (which will always be `nil`).